src/app/modules/myinfo/myinfo.constants.ts

```diff
@@ -15,6 +15,19 @@ export const MYINFO_ROUTER_PREFIX = '/myinfo'
  */
 export const MYINFO_REDIRECT_PATH = '/login'
 
+/**
+ * Name of cookie which passes the OAuth authorisation code
+ * from the /myinfo/login endpoint to the public form endpoint.
+ */
+export const MYINFO_AUTH_CODE_COOKIE_NAME = 'MyInfoAuthCode'
+
+/**
+ * Lifetime in ms of cookie containing auth code, which is used
+ * to pass the auth code from the /myinfo/login endpoint to the
+ * public form endpoint.
+ */
+export const MYINFO_AUTH_CODE_COOKIE_AGE_MS = 3 * 60 * 1000
```

Comment:
I wonder if this should be something shorter like 30 seconds instead of 3 minutes, since it should be used immediately. Longer expiry means there is a chance that if the user doesn't follow through on the redirect, someone else could use the auth code later on. But not a big deal tbh

Reply:
hmm yeah, I suppose it's just down to how slow we expect clients to be. I've seen really slow 3G connections before. I'll cut it to 2 minutes as a compromise but ultimately we're really just going for "as short as possible without disrupting slow users" and there's no correct answer I think haha

Reply:
2 minutes sounds good!

Reply:
eee75517